src/superqt/utils/_img_utils.py

```diff
@@ -37,4 +38,8 @@ def qimage_to_array(img: QImage) -> "np.ndarray":
     arr = np.frombuffer(b, np.uint8).reshape(h, w, c)
 
     # reverse channel colors for numpy
-    return arr.take([2, 1, 0, 3], axis=2)
+    # On big endian we need to specify a different order
+    if sys.byteorder == "big":
+        return arr.take([1, 2, 3, 0], axis=2)
```

Comment:
Should we add here  `  # pregma: no cover` to mark as not expected to be covered?

Reply:
Should I amend this or will you?